pkg/handler/unrouted_handler.go

```diff
@@ -17,7 +17,13 @@ import (
 )
 
 const UploadLengthDeferred = "1"
-const currentUploadDraftInteropVersion = "4"
+
+type DraftVersion string
+
+const (
+	Version3 DraftVersion = "3"
+	Version4 DraftVersion = "4"
```

Comment:
```suggestion
	interopVersion3 DraftVersion = "3" // From draft version -01
	interopVersion4 DraftVersion = "4" // From draft version -02
```

No need to export these constants.